src/identity-providers/add/DetailSettings.tsx

```diff
@@ -141,6 +158,31 @@ export const DetailSettings = () => {
   const isOIDC = providerId.includes("oidc");
   const isSAML = providerId.includes("saml");
 
+  const loader = async () => {
+    const loaderMappers = await adminClient.identityProviders.findMappers({
+      alias,
+    });
+
+    const loaderMapperTypes =
+      await adminClient.identityProviders.findMapperTypes({ alias });
+
+    const components = loaderMappers?.map((loaderMapper) => {
+      const provider = Object.values(loaderMapperTypes).filter(
```

Comment:
I believe `Object.values` can be removed here and the result will still be the same.
```suggestion
      const provider = loaderMapperTypes.filter(
```

Reply:
A `.filter is not a function` error gets thrown here unless `Object.values` is used